server/server.go

```diff
@@ -218,6 +218,14 @@ type Server struct {
 
 	// Websocket structure
 	websocket srvWebsocket
+
+	// susceptible is to mark whether this node is susceptible
+	// to dynamic cluster name INFO/CONNECT proto updates.
+	susceptible bool
```

Comment:
I am bad at naming things, but I am not sure that this is a good name. I think `clusterNameChangeAllowed` (although much longer) would be better.